tests/console/just_install_packages.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Basic installation of packages, using zypper. Packages separated by
+# space, defined using variable INSTALL_PACKAGES="apache2 krb5". In case of
+# installation failure, the module will fail.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use base 'consoletest';
+use warnings;
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    select_console 'root-console';
+    my $packages = get_var("INSTALL_PACKAGES");
```

Comment:
Instead of a variable it would be better to use test data and have a list of packages. We don't need to document yet another env variable.

Reply:
The variable already is used in other modules, I didn't notice that it's not documented. I can use test_data instead.